index.html

```diff
@@ -12592,6 +12832,97 @@ <h3>Serializing/Deserializing RDF</h3>
       </dd>
     </dl>
   </section>
+
+  <section class='informative'><h2>The `i18n` Namespace</h2>
+    <p>The `i18n:` namespace is used for describing combinations of <a>language tag</a> and <a>base direction</a> in <a>RDF literals</a>.
+      It is used as an alternative mechanism for describing the [[BCP47]] <a>language tag</a> and <a>base direction</a>
+      of <a>RDF literals</a> that would otherwise be use `xsd:string` or `rdf:langString` datatypes.</p>
+    <p>Datatypes based on this namespace allow round-tripping of JSON-LD documents using base direction,
+      although the mechanism is not otherwise standardized.</p>
+    <p>The <a data-cite="JSON-LD11-API#deserialize-json-ld-to-rdf-algorithm">Deserialize JSON-LD to RDF Algorithm</a>
+      can be used with the <dfn data-cite="JSON-LD11-API#dom-jsonldoptions-rdfDirection">rdfDirection</dfn>
+      set to `datatype` to generate <a>RDF literals</a> using the `i18n` base to create an IRI
```

Comment:
I would suggest something more specific, such as `i18n-datatype`, in case other implementation come up with alternative datatype-based solution